sentry/src/main/java/io/sentry/Hub.java

```diff
@@ -27,6 +27,7 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
+// TODO [HSM] remove Hub class
```

Comment:
I don't think anyone should be using `Hub` directly. It won't be supported anymore so we should just remove it.